charts/ccsm-helm/values.yaml

```diff
@@ -139,6 +138,10 @@ zeebe:
       cpu: 960m
       memory: 1920Mi
 
+  # RamdiskEnabled mounts a tmpfs (RAM-backed filesystem) for the data directory. Requires pvcEnabled: false. While tmpfs is very fast, be aware that unlike disks, tmpfs is cleared on node reboot and any files you write count against your container's memory limit.
+  ramdiskEnabled: false
+  # PvcEnabled if true, a persistent volume claim (PVC) is created for each broker of the Zeebe StatefulSet. CAVE: disabling persistence WILL lead to data loss, but can be preferable in a testing situation.
+  pvcEnabled: true
```

Comment:
:x: I think we can combine these. If pvcEnabled is false then it will be ram disk and vice versa

Reply:
Not realy, because there are three options. See my other comment on this PR.